crates/acir/src/circuit/gate.rs

```diff
@@ -43,7 +43,28 @@ impl Gate {
 #[derive(Clone, Debug)]
 /// Directives do not apply any constraints.
 pub enum Directive {
-    Invert { x: Witness, result: Witness },
+    Invert {
+        x: Witness,
+        result: Witness,
+    },
+    Quotient {
```

Comment:
Can we add comments above these to specify what they are doing?

For example, for invert, we can say:

"Inverts the value of x and stores it in the result variable"

Reply:
Done